src/layouts/docs/index.tsx

```diff
@@ -12,12 +13,16 @@ interface DocsLayoutProps {
   navData: MenuItem[]
   breadcrumbLinks?: BreadcrumbLink[]
   githubFileUrl?: string
+  backToLink?: {
+    text: string
+    url: string
+  }
```

Comment:
Added `backToLink` so that Figma mocks, which show different "Back to" links on `/waypoint` and `/waypoint/docs`, could be implemented.